examples/curator-s3-deployment/Furyfile.yml

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+bases:
+  - name: logging/curator-s3
+    version: v1.7.0
+  - name: logging/elasticsearch-single
+    version: v1.7.0
```

Comment:
Please, add this file to the bumpversion configuration file: https://github.com/sighupio/fury-kubernetes-logging/blob/master/.bumpversion.cfg in order to update the version hardcoded in this file on every new release